dev-libs/libmcrypt/libmcrypt-2.5.8.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="libmcrypt are intended to be replacements for the \
+old Unix crypt, except that they are under the GPL and \
+support an ever-wider range of algorithms and modes."
+HOMEPAGE="http://mcrypt.sourceforge.net/"
+COPYRIGHT="1998-2002 Nikos Mavroyanopoulos"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/mcrypt/Libmcrypt/$portVersion/libmcrypt-$portVersion.tar.gz"
```

Comment:
```diff
-SOURCE_URI="https://downloads.sourceforge.net/project/mcrypt/Libmcrypt/$portVersion/libmcrypt-$portVersion.tar.gz"
+SOURCE_URI="https://downloads.sourceforge.net/mcrypt/libmcrypt-$portVersion.tar.gz"
```

You should also add a
```
SECONDARY_ARCHITECTURES="x86"
```
as well as the magic `$secondaryArchSuffix` where appropriate.
This variable is:
* empty for primary arch builds, 
* equal to _x86 when you build mcrypt_x86 on x86_gcc2.

We also like to have some blank lines at some places, as described in:
[HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)